docs/sql/statements/create-table.rst

```diff
@@ -130,6 +130,9 @@ statement that doesn't contain an explicit value for it.
 The default clause :ref:`expression <gloss-expression>` is variable-free, it
 means that subqueries and cross-references to other columns are not allowed.
 
+.. NOTE::
+
+    Default values are not allowed for columns of type ``OBJECT``.
```

Comment:
AFAIK we allow default for known columns (`testCreateTableWithDefaultExpressionAsCompoundTypes`) - can we add for unknown columns? Maybe even smth like default expression cannot implicitly create dynamic columns?

Reply:
ObjectMapper is lacking completely support for `default_expr`, it's ignoring it and therefore ends up with an uknown entry in the properties Map. So, indeed it's supported in analysis, but not at runtime, it has no effect.

I thought it's better to just throw a message as a backport, and treat support for default clause as a feature. With a quick try, you can even have default values for subcols not defined in the `CREATE table` stmt, and they get an automatic type as if it was a DYNAMIC object adding a new subcolumn, but we need to add certain validations.

Reply:
ok, so it worked only on analysis, I got tricked by the test. Thx for clarifying!

Reply:
Can we instead log a warning in the backport?

Introducing errors in hotfix upgrades is not ideal.

Reply:
but we got an internal error anyway, that we need to somehow bypass if we choose to log a warn, or didn't I understand correctly?

Reply:
Oh nevermind. The comments above made me think this now adds an error where previously it was silently ignored.

The change is fine as is